pkg/server/tunnel.go

```diff
@@ -102,7 +102,21 @@ func (t *Tunnel) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	case <-connection.connected: // Waiting for response before we begin full communication.
 	}
 
-	defer conn.Close()
+	defer func() {
+		packet := &client.Packet{
+			Type: client.PacketType_CLOSE_REQ,
+			Payload: &client.Packet_CloseRequest{
+				CloseRequest: &client.CloseRequest{
+					ConnectID: connection.connectID,
+				},
+			},
+		}
+
+		if err = backend.Send(packet); err != nil {
+			klog.V(2).InfoS("failed to send close request packet", "host", r.Host, "agentID", connection.agentID, "connID", connection.connectID)
```

Comment:
Please move this to V(4). Failing to send a close request on tunnel shutting down isn't a surprise nor a significant state change. (Stopping the tunnel may be but the failure to send shutdown, less so)

Reply:
Talked to @Jefftree, and they convinced me we already downgraded the priority of this message once and to see how this priority plays out.